tslint.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
also, we should probably align this with the tslint config that's been evolving internally

Reply:
Agreed - It's a conversation that needs to happen!